### DISCUSSION TITLE
Feature request: more options for user input

### DISCUSSION THREAD

**Discussion by entusdictus** [jamf/Setup-Manager#6] [Ideas]
It would be beneficial to have more options around user input. 
For example, we're currently using several enrollment workflows where we use the "Department" field in Jamf Pro for scoping policies during device enrollment. This enables workflows where we use the same set of policy triggers for all enrollments, but with a different set of software at the end. Actually, there is no "department" value in this field, it's more of a "configuration" field in this example. Now, when in Setup Manager, it would be great if we have the option to rename the default "Department" label to something custom to match the workflow. Of course the functionality behind "department" wouldn't need to be changed - it's just a re-labeling of the field in the Setup Manager GUI. Of course this would apply to all user entry fields, not only department. 

In combination with the mentioned custom labeling above, a key like "waitForUserInput" would be really nice to ensure that the user has entered all necessary information before proceeding with policy execution. 

<img width="912" alt="Bildschirmfoto 2024-07-04 um 12 35 36" src="https://github.com/Jamf-Concepts/Setup-Manager/assets/14052665/57b3bab1-284b-4a40-9f13-e26cc2f6eef9">

**Comment by PhantomPhixer** [comment]
I just raised an FR for this and then found the discussions afterwards...
[FR](https://github.com/Jamf-Concepts/Setup-Manager/issues/40)
